client/src/components/Login.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import '../loginAndSignup.css';
+import { Link } from 'react-router-dom';
+
+export default function Login() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [message, setMessage] = useState('');
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      if (username === 'demo' && password === 'demo') {
+        window.location.href = '/';
```

Comment:
With ReactRouter it's best to use the `useNavigation` use instead of modifying `window.location.href`.